spec.html

```diff
@@ -435,6 +435,7 @@ <h1>CreateSegmentDataObject ( _segmenter_, _string_, _startIndex_, _endIndex_ )<
         1. Let _granularity_ be _segmenter_.[[SegmenterGranularity]].
         1. If _granularity_ is `"word"`, then
           1. Let _isWordLike_ be a Boolean value indicating whether the word segment _segment_ in _string_ is "word-like" according to locale _segmenter_.[[Locale]].
+          <emu-note>Whether a segment is "word-like" is implementation-dependent, and implementations are recommended to use locale-sensitive tailorings. In general, segments consisting solely of spaces and/or punctuation (such as those terminated with "WORD_NONE" boundaries by ICU [International Components for Unicode, documented at <a href="https://unicode-org.github.io/icu-docs/">https://unicode-org.github.io/icu-docs/</a>]) are not considered to be "word-like".</emu-note>
```

Comment:
Can you also add a pointer for this in Annex A?

https://tc39.es/ecma402/#annex-implementation-dependent-behaviour

Reply:
#102